tiup: add 3 docs disable/display/list (#5176)
pingcap/docs: tiup/tiup-component-cluster-list.md

---
title: tiup cluster list
---

# tiup cluster list

> tiup-cluster supports the deployment of multiple clusters using the same central control machine. You can use the `tiup cluster list` command to view which clusters are deployed by the currently logged-in user using this central control machine.

Reviewer: ```suggestion
tiup-cluster supports deploying multiple clusters using the same control machine. The `tiup cluster list` command outputs all clusters deployed by the currently logged-in user using this control machine.
```
`中控机` is `control machine`. You can check our Shimo term base.

> **Note:**
>

> > The deployed cluster data is placed in the `~/.tiup/storage/cluster/clusters/` directory by default, so on the same central control machine, the currently logged-in user cannot view the clusters deployed by other users.

Reviewer: ```suggestion
> The deployed cluster data is stored in the `~/.tiup/storage/cluster/clusters/` directory by default, so on the same control machine, the currently logged-in user cannot view the clusters deployed by other users.
```

## Syntax

```shell
tiup cluster list [flags]
```

## Options

### -h, --help

- Prints help information.
- Data type: `BOOLEAN`
- Default: false

## Outputs

Outputs the table with the following fields:

- Name: the cluster name
- User: the deployment user
- Version: the cluster version

> - Path: the path of the cluster deployment data on the central control machine

Reviewer: ```suggestion
- Path: the path of the cluster deployment data on the control machine
```

> - PrivateKey: the path of the private key that is connected to the cluster

Reviewer: ```suggestion
- PrivateKey: the path of the private key that is used to connect the cluster
```
